client/src/pages/searches/Search.js

```diff
@@ -802,7 +802,17 @@ const Search = ({
         </Alert>
       )}
       {queryTypes.includes(SEARCH_OBJECT_TYPES.ORGANIZATIONS) && (
-        <Fieldset id="organizations" title="Organizations">
+        <Fieldset
+          id="organizations"
+          title={
+            <>
+              Organizations
+              <Badge pill bg="secondary" className="ms-1">
```

Comment:
I suggest to only show the badge if there are search results (same as in the navigation sidebar). IMHO showing the `(0)` badge for "no objects found" is a bit distracting.

Reply:
Done